client/lcd/lcd_test.go

```diff
@@ -443,7 +443,7 @@ func TestSubmitProposal(t *testing.T) {
 
 	// query proposal
 	proposal := getProposal(t, port, proposalID)
-	require.Equal(t, "Test", proposal.Title)
+	require.Equal(t, "Test", proposal.GetTitle())
```

Comment:
Do you want to check the string or check that the enum is correct? If you check the latter, the string format can change later without breaking these tests.